README.md

```diff
@@ -1,5 +1,11 @@
 # libasynql <img src="https://poggit.pmmp.io/ci.badge/poggit/libasynql/libasynql" align="right"/>
 Asynchronous SQL access library for PocketMine plugins.
+## Why should I use this library and what asynchronous means?
+When excuting a SQL query at the main thread, **there will be a delay made** by MySQL for communicating between the database server or Sqlite for interacting with the file system. And the delay will completely block the main thread and **causes lag to the server**.
```

Comment:
```suggestion
When executing a SQL query on the main thread, **there will be a delay** to wait for the MySQL server or SQLite for interacting with the file system. The delay will block the main thread and **cause lag to the server**.
```

Reply:
Alright understable, I will "on threads" not "at threads"!